### PR TITLE
Speed up FR import process by merging INSERT INTO statements

### PR DESCRIPTION
**Pull request by vpodzime**
See individual commits for details.

### CODE REVIEW

**Comment by vpodzime** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2319/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2319/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2319/

File: configure.ac [hunk 1]
```diff
@@ -220,7 +220,7 @@ do
 done
 for i in templates cfe_internal
 do
-    for j in `find "$srcdir/$i" -name '*.mustache' -o -name '*.sh'`
+    for j in `find "$srcdir/$i" -name '*.mustache' -o -name '*.sh' -o -name '*.awk' -o -name '*.sed'`
```
Comment (craigcomstock) [inline configure.ac]: If we are making this nice change shouldn't we rename templates/federated_reporting/filter.sed.mustache to filter.sed?

File: cfe_internal/enterprise/federation/federation.cf [hunk 2]
```diff
@@ -287,6 +290,10 @@ bundle agent federation_manage_files
         perms => default:mog( "700", "root", "root" ),
         action => default:if_elapsed($(cfengine_enterprise_federation:config.dump_interval));
 
+      "$(cfengine_enterprise_federation:config.federation_dir)/fedhub/dump/filters/merge_inserts.awk"
```
[the remark this answers is not in the document]
Comment (vpodzime) [reply cfe_internal/enterprise/federation/federation.cf]: Yes, I even wanted to do that, thanks for reminding me!

File: configure.ac (hunk 1 above)
Comment (vpodzime) [reply configure.ac]: Yes, and we should also prefix with a number for ordering. Will do.

File: cfe_internal/enterprise/federation/federation.cf [hunk 3]
```diff
@@ -290,8 +290,8 @@ bundle agent federation_manage_files
         perms => default:mog( "700", "root", "root" ),
         action => default:if_elapsed($(cfengine_enterprise_federation:config.dump_interval));
 
-      "$(cfengine_enterprise_federation:config.federation_dir)/fedhub/dump/filters/merge_inserts.awk"
-        copy_from => default:local_dcp( "$(this.promise_dirname)/../../../templates/federated_reporting/merge_inserts.awk" ),
+      "$(cfengine_enterprise_federation:config.federation_dir)/fedhub/dump/filters/50-merge_inserts.awk"
```
Comment (nickanderson) [inline cfe_internal/enterprise/federation/federation.cf]: +1

File: templates/federated_reporting/dump.sh [hunk 4]
```diff
@@ -31,7 +31,7 @@ if ! type "$CFE_FR_COMPRESSOR" >/dev/null; then
 fi
 
 function sed_filters() {
-  sed_scripts="$(ls -1 "$CFE_FR_DUMP_FILTERS_DIR/"*".sed" 2>/dev/null)"
+  sed_scripts="$(ls -1 "$CFE_FR_DUMP_FILTERS_DIR/"*".sed" 2>/dev/null | sort)"
```
Comment (nickanderson) [inline templates/federated_reporting/dump.sh]: nice

**Comment by nickanderson** [comment]
Nice i like the numeric ordering.

**Pull request merged**